templates/emails/email-addresses.php

```diff
@@ -38,10 +38,10 @@
 			</address>
 		</td>
 		<?php if ( ! wc_ship_to_billing_address_only() && $order->needs_shipping_address() && ( $shipping = $order->get_formatted_shipping_address() ) ) : ?>
-			<td style="text-align:<?php echo $text_align; ?>; font-family: 'Helvetica Neue', Helvetica, Roboto, Arial, sans-serif; padding:0;" valign="top" width="50%">
-				<h2><?php _e( 'Shipping address', 'woocommerce' ); ?></h2>
+			<td style="text-align:<?php echo esc_html( $text_align ); ?>; font-family: 'Helvetica Neue', Helvetica, Roboto, Arial, sans-serif; padding:0;" valign="top" width="50%">
```

Comment:
`text-align:` should use `esc_attr()`, since is a attribute, `esc_html()` is not safe enough here.